app/konnect/runtime-manager/configuration/index.md

```diff
@@ -125,3 +125,34 @@ With **Keys**, you can centrally store and easily access key sets and keys in {{
 * PEM
 
 To learn more about the details of this feature, reference the [{{site.base_gateway}} key reference documentation](/gateway/latest/reference/key-management/). The {{site.konnect_short_name}} keys feature is built using {{site.base_gateway}} ability to manage keys, the documentation available can serve as a reference for both {{site.base_gateway}} and {{site.konnect_short_name}}.
+
+## Configuration limits
+
+The following limits apply to each runtime group for the configuration resources:
+
+| Resource Name | Limit |
```

Comment:
I think what your suggested change is valid; the other adjectives wouldn't be ideal though. `Service limits` could cause confusion as it would be an overloaded term since there is a service entity and `API request limits` is invalid since we aren't limiting the number of API requests.

Do you think we should remove the extra use of `resource` in the topic sentence?

```
The following entity resource limits apply to each runtime group for the configuration resources:
```

would become

```
The following entity resource limits apply to each runtime group for the configuration:
```